### PR TITLE
DTSPB-4201  change email to welsh 

### PR DESCRIPTION
**Pull request by abbasiTahir**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-4201

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -242,9 +242,9 @@ notifications:
       english:
         personal:
           applicationReceived: "6e3f4941-fd11-40fc-8480-8e314e85a985"
-          applicationReceivedNoDocs: "a6da6fb0-61a4-4a26-a324-805d7bcf733e"
+          applicationReceivedNoDocs: "39b7cef5-8202-479e-9f1e-86a76f964221"
           applicationReceivedPaperFormCaseworker: "01bfc52c-0850-4155-ae03-e6d638286288"
-          documentReceived: "24161fba-65f5-4384-9c4f-ef75f4a5c666"
+          documentReceived: "39b7cef5-8202-479e-9f1e-86a76f964221"
```
Comment (FeliTam) [inline src/main/resources/application.yml]: please do not amend the template for english notifications

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -290,9 +290,9 @@ notifications:
       welsh:
         personal:
           applicationReceived: "6e3f4941-fd11-40fc-8480-8e314e85a985"
-          applicationReceivedNoDocs: "a6da6fb0-61a4-4a26-a324-805d7bcf733e"
+          applicationReceivedNoDocs: "39b7cef5-8202-479e-9f1e-86a76f964221"
           applicationReceivedPaperFormCaseworker: "22c81233-a213-4520-b6bc-896fe8f6988a"
-          documentReceived: "24161fba-65f5-4384-9c4f-ef75f4a5c666"
+          documentReceived: "39b7cef5-8202-479e-9f1e-86a76f964221"
```
Comment (FeliTam) [inline src/main/resources/application.yml]: the template id should be different

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -290,9 +290,9 @@ notifications:
       welsh:
         personal:
           applicationReceived: "6e3f4941-fd11-40fc-8480-8e314e85a985"
```
Comment (FeliTam) [inline src/main/resources/application.yml]: where is the new template for Probate Application Received (personal)?

**Review by FeliTam — COMMENTED** [review]
Please add the corresponding Jira ticket number to the PR
Also, the same changes should apply to the application.yml in both /main folder and /contractTest folder

File: src/main/resources/application.yml (hunk 1 above)
Comment (abbasiTahir) [reply src/main/resources/application.yml]: resolved

File: src/main/resources/application.yml (hunk 2 above)
Comment (abbasiTahir) [reply src/main/resources/application.yml]: resolved

**Review by FeliTam — DISMISSED** [review]

**Review by FeliTam — APPROVED** [review]

**Pull request merged**